frontend/lib/norent/faqs.tsx

```diff
@@ -55,7 +51,16 @@ export const ChevronIcon = () => (
 );
 
 export const NorentFaqsPreview = () => {
-  const FaqsPreviewContent = FaqsContent.filter((faq) => faq.priority < 5);
+  const FaqsPreviewContent = FaqsContent.filter(
+    (faq) => faq.previewOptions
+  ).sort((faq1, faq2) =>
+    faq1.previewOptions?.priorityInPreview &&
+    faq2.previewOptions?.priorityInPreview
+      ? faq1.previewOptions.priorityInPreview -
+        faq2.previewOptions.priorityInPreview
+      : 0
+  );
+
```

Comment:
Agh so this definitely got a little messy here... it seemed that even after applying my `filter()` to grab faq entries that had their preview options configured, I'd still get a ts error staying that `previewOptions` could possibly be undefined. Any hints as to how I could avoid that?

Reply:
ahh yeah that is annoying that typescript's type inference doesn't totally work here.  I'm not sure of an easy way to work around it unfortunately, so maybe just add a comment mentioning what you wrote here, and we can figure out an easier way to write it out later?

Reply:
Sounds good to me! Thank you